app/controllers/events_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class EventsController < ApplicationController
+  def index
+    @events = Event.all
+  end
+
+  def show
+    @event = Event.find(params[:id])
+  end
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = current_user.events.build(event_params)
+    if @event.save
```

Comment:
let's flash something here to let the user know its done.
- on the form side of create, lets display the errors if it fails.